test/io-context-streaming.js

```diff
@@ -0,0 +1,121 @@
+const test = require('brittle')
+const ffmpeg = require('..')
+
+const { mediaTypes } = ffmpeg.constants
+
+test('IOContext streaming webm without onseek', (t) => {
```

Comment:
```suggestion
test('IOContext streaming webm with onread', (t) => {
```

It sounds more like what I see.